custom_components/tuya_local/translations/en.json

```diff
@@ -156,7 +156,10 @@
                     "mode": {
                         "state": {
                             "laundry": "Dry clothes",
-                            "purify": "Purify"
+                            "purify": "Purify",
```

Comment:
You should not need these changes.  The Home Assistant humidifier platform already defines auto, manual and sleep, and overriding them here will prevent the translations provided by Home Assistant from being found.

Reply:
I cannot find "manual" [here](https://developers.home-assistant.io/docs/core/entity/humidifier/#modes) – should I leave it?

Reply:
I think I used "normal" in other configs with the same options available.

Reply:
Ok, I'll change that then